processor/k8sattributesprocessor/README.md

```diff
@@ -272,10 +272,19 @@ k8sattributes/2:
       - k8s.node.name
       - k8s.pod.start_time
     labels:
-     # This label extraction rule takes the value 'app.kubernetes.io/component' label and maps it to the 'app.label.component' attribute which will be added to the associated resources
-     - tag_name: app.label.component
-       key: app.kubernetes.io/component
-       from: pod
+      # This label extraction rule takes the value 'app.kubernetes.io/component' label and maps it to the 'app.label.component' attribute which will be added to the associated resources
+      - tag_name: app.label.component
+        key: app.kubernetes.io/component
+        from: pod
+    operator_rules:
+      # Apply the operator rules - see https://github.com/open-telemetry/opentelemetry-operator#configure-resource-attributes
```

Comment:
I would avoid calling this feature `operator_rules`. If we decide on supporting this from the Collector at first place then it should have a generic name not coupled to the Operator project.

Also from this section it's not clear if we support both labels and annotations (as described in the Operator's docs). Is sth like `resource.opentelemetry.io/service.name: "my-service"` supported?

Reply:
> avoid calling this feature...

good idea - I just didn't find a good name so far

> annotations and labels

the linked page describes it: 

Annotations like this
```yaml
 annotations:
    # this is just an example, you can create any resource attributes you need
    resource.opentelemetry.io/service.name: "my-service"
    resource.opentelemetry.io/service.version: "1.0.0"
    resource.opentelemetry.io/deployment.environment.name: "production"
```

labels like this
```yaml
apiVersion: v1
kind: Pod
metadata:
  name: example-pod
  labels:
    app.kubernetes.io/name: "my-service"
    app.kubernetes.io/version: "1.0.0"
    app.kubernetes.io/part-of: "shop"
```

Reply:
https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/27261 suggests `well_known_labels`